tests/fixtures/common/collections/ansible_collections/testorg/coll_1/plugins/modules/mod_1.py

```diff
@@ -1,7 +1,11 @@
-"""An ansible test module."""
+"""An ansible test module.
+
+   isort:skip_file
```

Comment:
Can this be a comment? We need to avoid putting identifiers in docstrings. Or any other things that don't contribute to the explanation of what the module is about for that matter.

Reply:
I put it there as ansible-test still complained about it missing when it had a trailing comment. This appeared to be the next best place. Based on the docs, I saw only the 2 possible locations outside of the config: https://github.com/PyCQA/isort#skip-processing-of-imports-outside-of-configuration

Reply:
oh, I stand corrected, there are many ways: https://pycqa.github.io/isort/docs/configuration/action_comments.html

Reply:
I've skipped the entire collections folder in the configuration, since it just a fixture and all files would need to be excluded

Reply:
resolving conversation, please reopen if the concerns and question are unanswered, ty for the feedback